app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::API
+  include CanCan::ControllerAdditions
```

Comment:
@florencelau this was the one line that we needed to add to get the CanCan stuff to be recognized lol, since dream is an api-only rails app